### DISCUSSION TITLE
Do we have any way to handle and display file upload progress in Chopper?

### DISCUSSION THREAD

**Discussion by Purvik** [lejard-h/chopper#273] [Q&A] [answered]
As we're using `@multipart` annotation for API and `@partfile` parameter to have a file attached to upload file. Do we have any way to get progress of this task means how much bytes been uploaded so we can display some progress or percentage on screen?

**Comment by techouse** [comment, marked as answer]
https://github.com/lejard-h/chopper/issues/362
